dsl_parser/constants.py

```diff
@@ -97,7 +97,8 @@
 
 USER_PRIMITIVE_TYPES = ['string', 'integer', 'float', 'boolean', 'list',
                         'dict', 'regex', 'secret', 'textarea',
-                        'deployment_id', 'blueprint_id', 'capability_value']
+                        'deployment_id', 'blueprint_id', 'secret_key',
+                        'capability_value']
```

Comment:
This does change the index of capability_value- do we care about the positions of types in this list?

Reply:
not at all

Reply:
Excellent, I'll green then.